.buildkite/scripts/steps/unit-tests.sh

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
-set -uo pipefail
-
 source .buildkite/scripts/common.sh
+set +euo pipefail
```

Comment:
We set pipefail in the `common.sh` so the script was never run and performed the proper coverage files.

Reply:
What script was never run?

Reply:
Because of the order in which we set the bash flags, the' pipe fail` was overridden inside the `common.sh`, so the wrapping of the exit code was never run.